src/main.js

```diff
@@ -3,13 +3,23 @@ import InfoView from './view/main-info-view.js';
 import TripFilter from './view/trip-filters.js';
 import FormSort from './view/sort-main.js';
 import EventPresenter from './presenter/trip-event-presenter.js';
+import CreatePointList from './service/service.js';
+import ModelPoint from './model/point-model.js';
+import ModelOffer from './model/offer-model.js';
+import ModelDestinatios from './model/destination-model.js';
 
-const eventPresenter = new EventPresenter({listContainer: document.querySelector('.trip-events')});
+
+const mockService = new CreatePointList();
+const mockPoint = new ModelPoint(mockService);
+const mockOffer = new ModelOffer(mockService);
+const mockDestination = new ModelDestinatios(mockService);
+
+const eventPresenter = new EventPresenter({listContainer: document.querySelector('.trip-events')},mockPoint,mockOffer,mockDestination);
```

Comment:
Если функция принимает больше 3х параметров в большинстве случаев лучше завернуть в объект, для этого мы и сделали listContainer как свойство объекта:
```suggestion
const eventPresenter = new EventPresenter({
  listContainer: document.querySelector('.trip-events'),
  pointsModel: mockPoint,
  offersModel: mockOffer,
  destinationsModel: mockDestination
});
```
Это даст нам возможность точно передать то, что нужно, если, предположить, что аргументов будет 10, то очень сложно контролировать их порядок + опциональные аргументы, но заворачивая все в объект, мы можем указывать только то, что нужно, в любом порядке и явно

А на стороне презентера в той функции (методе) что мы получаем этот объект аргумент, мы просто деструктурируем эти свосйтва в переменные:
```ts
  constructor({listContainer, pointsModel, offersModel, destinationsModel}){

    this.listContainer = listContainer;
    this.pointsModel = pointsModel;
    this.offersModel = offersModel;
    this.destinationsModel = destinationsModel;
...
```
И опять же, мы можем и порядок изменить внутри и переименовать их как нужно и значения по умолчанию проставить -- плюсов гораздо больше.